core/base/base.go

```diff
@@ -155,10 +180,8 @@ func GetSeriesFromChannel(name string, channel string) (string, error) {
 func GetSeriesFromBase(v Base) (string, error) {
 	var osSeries map[SeriesName]seriesVersion
 	switch strings.ToLower(v.OS) {
-	case "ubuntu":
+	case UbuntuOS:
 		osSeries = ubuntuSeries
-	case "centos":
```

Comment:
I'm not sure we can drop centos in juju 3.x. In 4.0 yes. But last time I looked slurm and other HPC charms still used centos and we don't want to do such a breaking change until we have a major version bump.

Reply:
Oh yeah oops, I got confused about the version I was targeting